cernopendata/modules/fixtures/data/records/delphi-author-list-all.json

```diff
@@ -0,0 +1,4427 @@
+[
+  {
+    "abstract": {
+      "description": "DELPHI Collaboration authors"
+    },
+    "accelerator": "CERN-LEP",
+    "authors": [
+      [
+        {
+          "name": "Abdallah, J.",
+          "affiliation": "LPNHE, IN2P3-CNRS, Univ.~Paris VI et VII, Tour 33 (RdC), 4 place Jussieu, FR-75252 Paris Cedex 05, France"
+        },
+        {
+          "name": "Abreu, P.",
+          "affiliation": "LIP, IST, FCUL - Av. Elias Garcia, 14-$1^{o}$, PT-1000 Lisboa Codex, Portugal"
+        },
+        {
+          "name": "Adami, F.",
+          "affiliation": "DAPNIA - Service de Physique des Particules, CEA-Saclay, FR-91191 Gif-sur-Yvette Cedex, France"
+        },
+        {
+          "name": "Adam, W.",
+          "affiliation": "CERN, CH-1211 Geneva 23, Switzerland; Institut fuer Hochenergiephysik, Oesterr. Akad. d. Wissensch., Nikolsdorfergasse 18, AT-1050 Vienna, Austria"
+        },
+        {
+          "name": "A.Della~Riccia, B.",
```

Comment:
Is `A.Della~Riccia, B.` name recognised correctly? Should perhaps be `Della Riccia, B A`?

(judging from https://inspirehep.net/authors/2571661)